nerfstudio/models/splatfacto.py

```diff
@@ -103,7 +118,7 @@ class SplatfactoModelConfig(ModelConfig):
     """If True, continue to cull gaussians post refinement"""
     reset_alpha_every: int = 30
     """Every this many refinement steps, reset the alpha"""
-    densify_grad_thresh: float = 0.0002
+    densify_grad_thresh: float = 0.0008
```

Comment:
have we verified that this threshold results in roughly the same psnr as the standard grad threshold?

Reply:
This is the recommended setting in the paper. Based on my limited experiments so far, it creates less gaussians (~30% less) but comes with better quality. So the training is faster in general.